pkg/tf2pulumi/convert/testdata/partial_attribute_lookup/main.tf

```
@@ -1,4 +1,21 @@
 resource "simple_resource" "example" {
     input_one = data.unknown_data_source.example.attr
     input_two = unknown_resource_type.example.list[0]
+}
+
+resource "simple_resource" "another_example" {
+    input_one = example_resource_type.resource_name.some_attribute
+    input_two = var.example.test_attribute
+}
+
+output "test_unknown_already_declared_data_source" {
+    value = data.example.another_test_attribute
```

```suggestion
    value = data.unknown_data_source.example.another_test_attribute
```
to check that we use the same var as in line 2

---

Done ✅